### PR TITLE
Adding logo to navbar/ favicon 

### LINKED ISSUE #336: Enhancement: Add logo to PASS tab in web browser
**Describe the Current Behavior/Feature**:
Currently, when running PASS in a web browser, the tab doesn't display any particularly interesting icon. Just the browser default

image

**Proposed Behavior/Feature**:
We can display a nice icon here instead. A clean PASS logo in an SVG format would be perfect

**Proposed Implementation (if applicable)**:
To do this, all we need to do is drop a `favicon.ico` in the folder. W3 schools describes how to do this here:
https://www.w3schools.com/html/html_favicon.asp

### LINKED ISSUE #337: Enhancement: Update PASS header to have logo and be clickable
**Describe the Current Behavior/Feature**:
Currently in the navbar, we have the word PASS in the upper left corner of the navbar. It's in the same space as an App logo is normally. However, no logo is present, and unlike many apps, the text does not navigate you back to the home screen.

image

**Proposed Behavior/Feature**:
Replace the PASS text with a clickable icon that navigates you back to a home page in the app. Similar to how the github logo navigates you to the github home page.

**Rationale**:
A logo will make the app look more professional, and a clickable logo is a common feature in many websites that users will probably be familiar with.

### PR DESCRIPTION
**Pull request by Raspber**
Added favicon.png to the asset folder and the Link to the icon in the index.html file.
Replaced the "PASS" text with an img tag that includes the logo from the asset folder.

### CODE REVIEW

**Review by timbot1789 — COMMENTED** [review]
This looks good to me! Do you think you could delete the old logo.png in the asset folder too?

**Comment by Raspber** [comment]
i deleted the old logo from the asset folder and I also fixed the lint issues

File: src/assets/logo.png
Comment (timbot1789) [inline src/assets/logo.png:1]: I always love seeing -60 kb

File: src/components/NavBar/NavbarDesktop.jsx [hunk 1]
```diff
@@ -61,9 +60,7 @@ const NavbarDesktop = ({ setShowConfirmation }) => {
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static" color="primary">
         <Toolbar sx={{ minHeight: '64px' }}>
-          <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
-            PASS
-          </Typography>
+        <img src='src/assets/pass-logo.png' alt='logo' className='navbar-logo'/>
```
Comment (timbot1789) [inline src/components/NavBar/NavbarDesktop.jsx]: It looks like a unit test is failing because you removed the `aria-label` attribute. The test is looking for that label to figure out where the logo is, and make sure the navbar is rendering properly. Could you add that attribute to this component?

**Review by timbot1789 — COMMENTED** [review]
Something I missed the first go-around, but the `NavBarMobile.jsx` component also has a logo in it. Can you put the `img` there as well? Should be identical to the Desktop and LoggedOut version.

File: src/style.css [hunk 2]
```diff
@@ -33,3 +33,8 @@ header {
   padding-left: 1rem;
   color: purple;
 }
+
```
Comment (xscottxbrownx) [inline src/style.css]: we are trying to get rid of the `style.css` file, by switching the project to Material UI.

For now, could you just add this inline style in the components instead? `style={{ marginRight: '2rem' }}`

---

We'll need to discuss images as a team. MUI doesn't have an image component, they recommend another library dependency: https://mui.com/material-ui/discover-more/related-projects/#image

**Comment by andycwilliams** [comment]
Looks like you're trying to merge from a forked version of PASS rather than a separate branch within this repo. Not insurmountable, but it makes things a bit smoother to just create a branch instead.

Also a naming convention is to state what's specifically being addressed in the title, usually including an associated issue number. So instead of "Hauvert pass" it would be something like "251/update inactivity message" or "Add new user registration flow #332". It helps keep things organized.

File: src/assets/logo.png
Comment (leekahung) [reply src/assets/logo.png:1]: logo.png is being used for the PASS logo in the GitHub README.md file. Could we perhaps get a replacement image for it using your new logo or update README.md to exclude the image? Either would be fine.

File: src/components/NavBar/NavbarMobile.jsx [hunk 3]
```diff
@@ -57,9 +57,7 @@ const NavbarMobile = ({ setShowConfirmation }) => {
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static" color="primary">
         <Toolbar sx={{ minHeight: '64px' }}>
-          <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
```
Comment (xscottxbrownx) [inline src/components/NavBar/NavbarMobile.jsx:60]: Can now remove the `<Typography>` import too.

File: src/style.css (hunk 2 above)
Comment (xscottxbrownx) [reply src/style.css]: > we are trying to get rid of the `style.css` file, by switching the project to Material UI.
> 
> For now, could you just add this inline style in the components instead? `style={{ marginRight: '2rem' }}`


This still

File: src/components/NavBar/NavbarDesktop.jsx [hunk 4]
```diff
@@ -61,9 +60,7 @@ const NavbarDesktop = ({ setShowConfirmation }) => {
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static" color="primary">
         <Toolbar sx={{ minHeight: '64px' }}>
-          <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
-            PASS
-          </Typography>
+        <img src='src/assets/pass-logo.png' alt='logo' className='navbar-logo'aria-label="logo"/>
```
Comment (xscottxbrownx) [inline src/components/NavBar/NavbarDesktop.jsx]: see other comment, but do styling inline for now. we are trying to get rid of `styles.css` file.

File: src/components/NavBar/NavbarLoggedOut.jsx [hunk 5]
```diff
@@ -20,9 +19,7 @@ const NavbarLoggedOut = () => (
   <Box sx={{ flexGrow: 1 }}>
     <AppBar position="static" color="primary">
       <Toolbar sx={{ minHeight: '64px' }}>
-        <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
-          PASS
-        </Typography>
+      <img src='src/assets/pass-logo.png' alt='logo' className='navbar-logo' aria-label="logo"/>
```
Comment (xscottxbrownx) [inline src/components/NavBar/NavbarLoggedOut.jsx]: see other comment, but do styling inline for now. we are trying to get rid of `styles.css` file.

File: src/components/NavBar/NavbarMobile.jsx [hunk 6]
```diff
@@ -57,9 +57,7 @@ const NavbarMobile = ({ setShowConfirmation }) => {
...
-            PASS
-          </Typography>
+          <img src='src/assets/pass-logo.png' alt='logo' className='navbar-logo' aria-label='logo'/>
```
Comment (xscottxbrownx) [inline src/components/NavBar/NavbarMobile.jsx]: see other comment, but do styling inline for now. we are trying to get rid of `styles.css` file.

**Review by xscottxbrownx — CHANGES_REQUESTED** [review]

File: src/components/NavBar/NavbarDesktop.jsx (hunk 4 above)
Comment (Raspber) [reply src/components/NavBar/NavbarDesktop.jsx]: sorry i just seen this now. I will get do inline styling.

File: src/components/NavBar/NavbarMobile.jsx (hunk 3 above)
Comment (Raspber) [reply src/components/NavBar/NavbarMobile.jsx:60]: I believe i deleted this. Just need to remove the css and do inline styling.

File: src/components/NavBar/NavbarMobile.jsx (hunk 6 above)
Comment (Raspber) [reply src/components/NavBar/NavbarMobile.jsx]: hey scott, i believed i made all required changes.

File: index.html [hunk 7]
```diff
@@ -5,6 +5,8 @@
     <meta name="viewport" content="initial-scale=1, width=device-width" />
     <title>Getting Started with PASS</title>
     <link rel="stylesheet" href="/src/style.css" />
+    <link rel="icon" type="image/x-icon" href="src/assets/pass-logo.png">
+
```
Comment (leekahung) [inline index.html]: I don't think you need an extra space here.

File: package.json
Comment (leekahung) [inline package.json:1]: For `package.json`, I think you could revert these changes and have your branch updated instead since the latest version of Development already have these fixes in place.

**Comment by xscottxbrownx** [comment]
what are the `.zip` files in `src/assets`?

File: src/assets/ceha-search-message-dark-mode-app-ui-2023-05-09-21-21-10-utc.zip
Comment (leekahung) [inline src/assets/ceha-search-message-dark-mode-app-ui-2023-05-09-21-21-10-utc.zip:1]: A bit confused about these zip files. What are they used for?

Comment (Raspber) [reply src/assets/ceha-search-message-dark-mode-app-ui-2023-05-09-21-21-10-utc.zip:1]: I dont see them. I think I may have saved folders in the asset folder by accident. I do not see them in there anymore.

Comment (leekahung) [reply src/assets/ceha-search-message-dark-mode-app-ui-2023-05-09-21-21-10-utc.zip:1]: If that's the case,  could you remove these zip files from git using `git rm`?

**Review by timbot1789 — APPROVED** [review]

**Review by xscottxbrownx — APPROVED** [review]
A couple of the NavBar files have imports that are now not in alphabetical order, but it's not an issue.    Just keep in mind with future updates, we've been trying to keep each "category" of imports alphabetized so we can find what we need faster.

File: index.html [hunk 8]
```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="initial-scale=1, width=device-width" />
+    <meta charset="utf-8"/>
```
Comment (leekahung) [inline index.html]: I'm a little confused over these changes with the spaces inside the meta and link tags in Line 4, 5, 7, and 8? I think we could leave them alone.

**Review by leekahung — CHANGES_REQUESTED** [review]
Mostly good with everything, just confused over one thing. See feedback.

**Comment by leekahung** [comment]
Hey @Raspber. How's the file changes going? I believe it's just those missing spaces, then I should be able to approve the PR for merger.

File: index.html [hunk 9]
```diff
@@ -1,10 +1,10 @@
...
 <html>
   <head>
-    <meta charset="utf-8"/>
-    <meta name="viewport" content="initial-scale=1, width=device-width"/>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="initial-scale=1, width=device-width" />
     <title>Getting Started with PASS</title>
-    <link rel="stylesheet" href="/src/style.css"/>
+    <link rel="stylesheet" href="/src/style.css" />
     <link rel="icon" type="image/x-icon" href="src/assets/pass-logo.png">
```
Comment (leekahung) [inline index.html]: I've just noticed, but the link in Line 8 is missing a self-closing tag `/>` when compared to the lines above it.

File: index.html [hunk 10]
```diff
@@ -5,8 +5,8 @@
     <meta name="viewport" content="initial-scale=1, width=device-width" />
     <title>Getting Started with PASS</title>
     <link rel="stylesheet" href="/src/style.css" />
+    <link rel="icon" type="image/x-icon" href="src/assets/pass-logo.png"/>
```
Comment (leekahung) [inline index.html:8]: It's still missing the space for it, but it's fine. We can clean it up later with prettier or the like. I'll approve for now.

**Review by leekahung — APPROVED** [review]
Think it's ready for merger.
